scalableminds/webknossos
# add button to clear data cube cache for dataset

### URL of deployed dev instance (used for testing):
- https://flushdatacubecache.webknossos.xyz

### Steps to test:
- data should still load
- try new datastore route `/data/triggers/clearCache/<dataSetName>?token=<accessToken>`
- data should be loaded from file again after that

### Issues:
- fixes #2754 

------
